src/SearchEndpoint/AggregationsEndpoint.php

```diff
@@ -27,7 +27,7 @@ class AggregationsEndpoint extends AbstractSearchEndpoint
     /**
      * {@inheritdoc}
      */
-    public function normalize(NormalizerInterface $normalizer, string $format = null, array $context = [])
+    public function normalize(NormalizerInterface $normalizer, string $format = null, array $context = []): array|string|int|float|bool
```

Comment:
Union return types are from php 8 and in composer we declare that php 7.4 is supported.

Reply:
I think it's time to remove 7.4 as it's no longer supported :(

https://www.php.net/supported-versions.php

Or we can return nothing in this function